src/main/scala/fairysf/variant/MiniBreakthroughTroyka.scala

```diff
@@ -0,0 +1,42 @@
+package strategygames.fairysf
+package variant
+
+import strategygames.GameFamily
+import strategygames.Player
+
+case object MiniBreakthroughTroyka
+    extends Variant(
+      id = 10,
+      key = "minibreakthroughtroyka",
+      name = "Mini Breakthrough",
+      standardInitialPosition = true,
+      fairysfName = FairySFName("ps-minibreakthrough"),
+      boardSize = Board.Dim5x5
+    ) {
+
+  def gameFamily: GameFamily = GameFamily.BreakthroughTroyka()
+
+  def perfIcon: Char                      = ''
+  def perfId: Int                         = 209
+  override def canOfferDraw               = false
+  override def repetitionEnabled: Boolean = true
```

Comment:
why repetitionEnabled, dont think that is possible in this variant anyways therefore dont think we need this?